ocaml/xenopsd/scripts/setup-vif-rules

```diff
@@ -196,7 +193,7 @@ def clear_xs_ofport_path(vif):
 
 def make_vswitch_external_ids(vif):
     args = []
-    for (k, v) in vif.get_external_ids().items():
+    for (k, v) in list(vif.get_external_ids().items()):
```

Comment:
I know the return type of `{}.items()` are different, 
but are't both of them are iterable?  is this `list` necessary?  (but has not bad anyway, just curious)

Reply:
Yes, you're right. It's not necessary. :)

Reply:
I think you mean line 95:
def do_chain_action(executable, action, chain_name, args=[]):